1_historical_RS_data_collation.R

```diff
@@ -0,0 +1,87 @@
+# Source functions for this {targets} list
+tar_source("1_historical_RS_data_collation/src/")
+
+# Download and process GEE output from historical pulls -------------
+
+# this pipeline collates all of the GEE output files for the NW and CLP projects
+# Note: this portion of the workflow is dependent on the successful run of two 
+# branches of the Landsat_C2_SRST repository: nw-poudre-historical and nw-er3z21-historical.
+# At this time, this is run outside of the {targets} workflow presented here. 
+
+p1_targets_list <- list(
+  # download the NW and CLP data from Google Drive
+  tar_target(
+    name = p1_download_historical_NW_CLP,
```

Comment:
suggestion: I typically name functions with verbs (e.g., `download`) and targets with past participles (e.g., `downloaded`) to give a sense for what the completion of the target indicates in the pipeline. I'm sure this is open to preference, and I think that I got this from the {targets} or {drake} user guide early on?

Reply:
Thank you for this suggestion! I've been trying to incorporate the function-verb naming scheme, but the past participle for the target is an amazing pointer. I'll go back and check this in the next round.